docker-compose.yml

```diff
@@ -281,10 +284,39 @@ services:
         tag: "desec/openvpn-server"
     restart: unless-stopped
 
+  prometheus:
+    image: prom/prometheus:latest
+    init: true
+    volumes:
+    - ./prometheus/conf:/etc/prometheus:ro
+    - prometheus:/prometheus
```

Comment:
is there a difference in `./prometheus/` (two lines above) and `prometheus/` (line above)?

Reply:
Yes! `prometheus` is the volume defined further down in the file, and `./prometheus` is a directory relative to the current directory. The volume carries persistent data, the other one is for mounting configuration.